dev/build/makepack-howto.md

```diff
@@ -0,0 +1,142 @@
+# Dolibarr Makepack How To
+
+This documentation describe steps to build a BETA or RELEASE versions of Dolibarr.
+There is a chapter for BETA version and a chapter for RELEASE version.
+
+
+## Prerequisites on Linux
+
+Prerequisites to build the tgz, debian and rpm packages:
+
+`apt-get install perl tar dpkg dpatch p7zip-full rpm zip php-cli`
+
+Prerequisites to build autoexe DoliWamp package from Linux (solution seems broken since Ubuntu 20.04+):
+
+`apt-get install wine q4wine`
+
+* Launch "wine cmd" to check a drive Z: pointing to / exists.
```

Comment:
I recommend to keep `*` for highlighting and `-` or numbers (`1.`) for lists.  These are also the values proposed by the github comment editor and values that `mdformat` uses.